### PR TITLE
specific changes for Mac users

### PR DESCRIPTION
**Pull request by HeeroML**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /.idea/
+.DS_Store
```
Comment (roj1512) [inline .gitignore:2]: I'm doubtful, is that .DS_Store or .DS_STORE? And why wouldn't it get ignored by default anyway?

Comment (HeeroML) [reply .gitignore:2]: i don't know, it didnt and its annoying to remove it every time :D Its, `.DS_Store`

**Review by roj1512 — APPROVED** [review]
Well, fine. Let's show some mercy for those appleboys out there.

**Comment by KnorpelSenf** [comment]
What a peculiar change. Is the apple stuff incapable of deactivating these files that pollute git repositores?

**Comment by HeeroML** [comment]
VS Code and git itself add them all the time.

**Comment by KnorpelSenf** [comment]
Aren't those system files?

**Comment by HeeroML** [comment]
Yes

**Pull request merged**